src/main/java/frc/robot/subsystems/ClimberSubsystem.java

```diff
@@ -101,8 +101,11 @@ public boolean isAvailable()
      /*
      * Returns the singleton instance for the Climber Subsystem
      */
-    public static ClimberSubSystem getInstance() {
-      return INSTANCE_CLIMBER;
+    public static ClimberSubsystem getInstance() {
```

Comment:
Why do you need this?  isAvaliable() is just checking if the climber != null.   If it is null it will return null anyway.  The code that was already there was more efficient.

Reply:
The INSTANCE_CLIMBER will not be null, but the m_climber has a possibility of being null.